packages/discovery-provider/plugins/notifications/src/email/notifications/preRendered/purchase.ts

```diff
@@ -1,5 +1,8 @@
+import { capitalize } from 'lodash'
```

Comment:
Just asking since this is the only one. Are we allowed to import packages here?

I assume it's fine cause this is just a discovery-provider plugin

Reply:
it is just a ts file so i would assume it's ok..

Reply:
should be fine